src/rgw/store/dbstore/common/dbstore.cc

```diff
@@ -373,6 +373,7 @@ int DB::store_user(const DoutPrefixProvider *dpp,
   if (ret) {
     ldpp_dout(dpp, 0)<<"store_user failed with err:(" <<ret<<") " << dendl;
     goto out;
+    ldpp_dout(dpp, 20)<<"User creation successful - userid:(" <<uinfo.user_id<<") " << dendl;
```

Comment:
this line is unreachable due to the `goto` immediately above. looks like this belongs outside the `}` below

Reply:
Fixed it.